swap_meet/electronics.py

```diff
@@ -1,2 +1,14 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+    def __init__(self, id=None, type="Unknown", condition=0.0, age=0):
+        super().__init__(id, condition)
```

Comment:
Nice, this is what the other children classes should be doing, minus the redundant `self.condition = condition`. You are already setting the `condition` for the item with the `__init__` function from `Item`.